alicloud/resource_alicloud_slb.go

```diff
@@ -1,12 +1,17 @@
 package alicloud
 
 import (
+	"bytes"
+	"errors"
 	"fmt"
+	"reflect"
+	"strings"
+	"time"
 
+	"github.com/denverdino/aliyungo/common"
 	"github.com/denverdino/aliyungo/slb"
```

Comment:
I think line 4, 5, 7, 8 and 11 should be removed. These are useless code in the latest master branch.